source/falaise/snemo/services/dead_cells.h

```diff
@@ -0,0 +1,128 @@
+//! \file falaise/snemo/services/dead_cells.h
+//! \brief Tracker Dead Cells service in SuperNEMO
+//
+// more doxygen to come...
+//
+// status  description
+// ---------------------------------------------------------------------------
+// 0       good working cell            isAGoodCell==true
+// 1       dead cells with dead anode   isADeadCell==true and isABadCell==true
+// 2       cathode-to-ground short      isABadCell==true
+// 3       cathode-to-cathode short     isABadCell==true
+// 4       other                        isABadCell==true
+
+
+#ifndef SNEMO_DEAD_CELLS_SVC_HH
+#define SNEMO_DEAD_CELLS_SVC_HH
+
+#include <iostream>
+#include <fstream>
+
+#include "bayeux/datatools/base_service.h"
+#include "falaise/snemo/services/service_traits.h"
+#include "falaise/bounded_int.h"
+
+namespace snemo {
+
+  // data member for the possible statuses of a cell  
+  enum status {
+    good = 0,            // good working cell
+    dead = 1,            // dead cells with dead anode
+    cathode_ground = 2,  // cathode-to-ground short
+    cathode_cathode = 3, // cathode-to-cathode short
+    other = 4            // other
+  };
```

Comment:
```suggestion
  enum class cell_status {
    good = 0,            // good working cell
    dead = 1,            // dead cells with dead anode
    cathode_ground = 2,  // cathode-to-ground short
    cathode_cathode = 3, // cathode-to-cathode short
    other = 4            // other
  };
```

Should be an `enum class` to give stronger typing and clarity.

Reply:
> Ah @emchauve , we crossed over in our timings there! I think where we have a cathode-to-cathode short, we should identify whether it's at the top or bottom (we have this info already) and then just not activate those particular cathodes. But the rest of the cell should be fine.

ok I will add  for completeness `cathode_ground_top`, `cathode_ground_bottom` and `cathode_ground_both`